packages/typespec-autorest/src/openapi.ts

```diff
@@ -1801,7 +1802,7 @@ export async function getOpenAPIForService(
     return getSchemaForType(variant.type, schemaContext)!;
   }
 
-  function getDefaultValue(defaultType: Value): any {
+  function getDefaultValue(defaultType: Value, modelProperty?: ModelProperty): any {
```

Comment:
same as in openapi3, I think the whole function content can just be calling to serializeValueAsJson without anything else

Reply:
I don't think is the case, I replied with more information on the openapi3 comment, but I will reach you out so we can chat about this :)